catt/stream_info.py

```diff
@@ -49,7 +49,10 @@ def __init__(self, video_url, model=None, host=None):
             if self.is_playlist:
                 items = list(self._preinfo["entries"])
                 self._first_entry_info = self._get_stream_info(items[0])
-                self._playlist_items = [item["id"] for item in items]
+                try:
+                    self._playlist_items = [item["id"] for item in items]
+                except KeyError:
```

Comment:
Can you add a comment here, explaining why you're doing this? You're building Chesterton's fence here.

Reply:
Sure.

Reply:
Great, I'm cool to merge after that, thanks!